makePDF.py

```diff
@@ -75,7 +84,167 @@ def getOEMCP():
     return str(codepage)
 
 
+class LatextoolsBuildSelector(sublime_plugin.WindowCommand):
+
+	# on 3080+, the necessary features are built-in by default
+	if sublime.version() > '3079':
+		def run(self, select=False):
+			select = False if select not in (True, False) else select
+			self.window.run_command('build', {'select': select})

```

Comment:
I think it would be better to use a context in the keybinding and write an EventListener. There may be packages, which map the `ctrl+b` to something else, e.g. Sublimemacs.